asset/quickstart/analyze-iam-policy-longrunning-bigquery/main_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestMain(t *testing.T) {
+	tc := testutil.SystemTest(t)
```

Comment:
How long is this test expected to take? If it's more than a few seconds, will you use `testutil.EndToEndTest` to make it so it only runs in one Go version and in nightly jobs? Same with other samples.

Reply:
It will take one second or two, so I guess we are ok.